### PR TITLE
cp-423: improvements in dropdown  burger menu

### PR DESCRIPTION
**Pull request by Xxandder**
- Fixed the wrong view of home item when it was active
- Increased the gap between icons and text

image

### CODE REVIEW

File: frontend/src/libs/components/dropdown-menu/styles.module.scss [hunk 1]
```diff
@@ -74,11 +74,6 @@
     brightness(93%) contrast(86%);
 }
 
-.selected {
-  background-color: var(--gray-100);
-  cursor: pointer;
-}
-
```
Comment (dmytro-kucherenko) [inline frontend/src/libs/components/dropdown-menu/styles.module.scss]: Should the selected option highlighting be removed totally? I think user have to see that and the problem was with different `pathname` due to the url params

Comment (v0ldemar01) [reply frontend/src/libs/components/dropdown-menu/styles.module.scss]: Good point

Comment (Xxandder) [reply frontend/src/libs/components/dropdown-menu/styles.module.scss]: Yep, thanks for the noticing. I will do it in this way

**Review by fshabanov — DISMISSED** [review]

File: frontend/src/libs/components/dropdown-menu/dropdown-menu.tsx [hunk 2]
```diff
@@ -11,15 +12,28 @@ type Properties = {
...
 
+  const checkIsCurrentRoot = useCallback(
+    (currentPath: string, itemPath: string): boolean => {
+      const currentPathWithoutSlash = currentPath.slice(SECOND_INDEX);
+      const itemPathWithoutSlash = itemPath.slice(SECOND_INDEX);
+
+      return (
+        currentPath === itemPath ||
+        (currentPathWithoutSlash !== '' &&
+          itemPathWithoutSlash.startsWith(currentPathWithoutSlash))
```
Comment (v0ldemar01) [inline frontend/src/libs/components/dropdown-menu/dropdown-menu.tsx]: We already have helper for it for sidebar

**Review by v0ldemar01 — CHANGES_REQUESTED** [review]

**Review by OlenaYatseniuk — APPROVED** [review]

**Review by liza-veis — APPROVED** [review]

**Pull request merged**